.github/workflows/tag.yml

```diff
@@ -8,13 +8,13 @@ on:
 jobs:
   build-n-publish:
     name: Build and publish Python 🐍 distributions 📦 to PyPI
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Just confirming we want to be on a rolling release here, and not pinned to a specific version?  

I'm ok with this, and we should definitely update our pin from 18.04 regardless, but just making sure we're comfortable with the tradeoffs of not pinning a specific version.